kolibri/plugins/learn/assets/src/views/LearnImmersiveLayout.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+
+  <div
+    v-if="!loading"
+    ref="mainWrapper"
+    class="main-wrapper"
+  >
+
+    <div v-if="blockDoubleClicks" class="click-mask"></div>
+    <SkipNavigationLink />
+    <LearningActivityBar
+      :resourceTitle="resourceTitle"
+      :learningActivities="mappedLearningActivities"
+      :isLessonContext="true"
+      :isBookmarked="true"
+      :allowMarkComplete="false"
+      data-test="learningActivityBar"
+      @navigateBack="navigateBack"
+    />
+    <LessonMasteryBar
```

Comment:
Mastery and hints are specific to exercises, so I feel like this might better live inside the `AssessmentWrapper` component, rather than at the top level of the immersive layout here.

If the intention is to make this LearnImmersiveLayout more general than just for housing the content page, then I think that gives even more reason to do so - if it is meant to be just for content, we can probably consolidate more of the `ContentPage` component into this component.